### PR TITLE
Nickfix

### PR DESCRIPTION
**Pull request by opoudjis**

### CODE REVIEW

File: scripts/split_codes.rb [hunk 1]
```diff
@@ -19,7 +19,10 @@ def split(input, output)
     end
 
     def swap_term_refs(string)
-      string.gsub(/\{\{\s*([^}]+)\s*,\s*([^}]+)\s*\}\}/, '{{<<\2>>,\1}}')
+      # string.gsub(/\{\{\s*([^}]+)\s*,\s*([^}]+)\s*\}\}/, '{{<<\2>>,\1}}')
+      string.gsub(/\{\{\s*([^}]+)\s*,\s*([^}]+)\s*\}\}/, '{{<<\2>>}}')
+            .gsub(/\(\{\{([^}]+)\}\}\}/, '{{\1}}')
```
Comment (ronaldtse) [inline scripts/split_codes.rb]: There's an issue with the replacement, the brackets are not matching.

File: scripts/split_codes.rb [hunk 2]
```diff
@@ -29,7 +32,11 @@ def fix_image_paths(string)
     end
 
     def fix_github_issues(string)
-      swap_term_refs(fix_image_paths(string))
+      refs_patch(swap_term_refs(fix_image_paths(string)))
+    end
+
+    def refs_patch(string)
+      string.gsub(/IEC 60050-191:1990/, '<<IEV191_1990>>')
```
Comment (ronaldtse) [inline scripts/split_codes.rb:39]: Why not replace with the generic IEV anchor instead of an individual 60050 part?

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: scripts/split_codes.rb (hunk 1 above)
Comment (opoudjis) [reply scripts/split_codes.rb]: Fixed

File: scripts/split_codes.rb (hunk 2 above)
Comment (opoudjis) [reply scripts/split_codes.rb:39]: These are not term references, they are references to the entire IEV part document. A generic IEV anchor only works if provided a full term reference.

Comment (ronaldtse) [reply scripts/split_codes.rb:39]: OK

**Review by ronaldtse — APPROVED** [review]

**Pull request merged**